js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -1935,7 +1935,7 @@ scopePermissions.groups.view-description=Policies that decide if an administrato
 tokens=Tokens
 createFlow=Create flow
 encryptAssertionsHelp=Should SAML assertions be encrypted with client's public key?
-oAuthDPoPHelp=This enables support for Demonstrating Proof-of-Possession (DPoP) bound tokens. The access and refresh tokens are bound to the key stored on the user agent. In order to prove the possession of the key, the user agent must send a signed proof alongside the token.
+oAuthDPoPHelp=This enables support for Demonstrating Proof-of-Possession (DPoP) bound tokens. The access and refresh tokens are bound to the key stored on the user agent. In order to prove the possession of the key, the user agent must send a signed proof alongside the token. DPoP can only be configured for public clients.
```

Comment:
This is not accurate. DPoP can be configured for both public and confidential clients. The only thing is, that for confidential clients, only access-tokens are DPoP bound. For public clients, both access-tokens and refresh-tokens are DPoP bounds. In other words, the binding of refresh tokens is only for public clients.

Is it possible to update the tooltip this way and make sure that switch is visible for both public and confidential clients?

Also you can possibly either update docs or revert changes of docs from this PR (as there is dedicated task for docs, which I am planning to handle once other PRs are done).

Reply:
Ah! Then it should not be hidden for confidential. I'll do it after lunch.